### DISCUSSION TITLE
Can't boot  3.1.0 on Ubuntu 16.04.1 LTS

### DISCUSSION THREAD

**Discussion by jaimeagudo** [apache/couchdb#2963] [Q&A]
## Description
Can't boot installation 

## Steps to Reproduce
* Regular installation using apt on Ubuntu 16.04.1 LTS as per docs.
* Stuck at https://docs.couchdb.org/en/stable/install/unix.html#first-run
* I follow https://docs.couchdb.org/en/stable/install/troubleshooting.html and I am stuck at

```
erl -env ERL_LIBS $ERL_LIBS:/home/couchdb/lib -couch_ini -s crypto                                                                                           127
Erlang/OTP 18 [erts-7.3] [source] [64-bit] [async-threads:10] [kernel-poll:false]

{"init terminating in do_boot",{undef,[{crypto,start,[],[]},{init,start_it,1,[]},{init,start_em,1,[]}]}}

Crash dump is being written to: erl_crash.dump...done
init terminating in do_boot ()

```
There is a ` erl_crash.dump` which I am happy to share cause I honestly have no clue of Erlang

This answer made no difference https://stackoverflow.com/questions/23619981/couchdb-init-terminating-in-do-boot-error-in-ubuntu-14-04

## Expected Behaviour

Couchdb boots

## Your Environment

* CouchDB version used: 3.1.0
* Browser name and version: N/A
* Operating system and version:   Ubuntu 16.04.1 LTS  on Linode if that makes a difference connected through SSH
* There was a 9 month old version of couchdb installed but not setup (2.x, sorry I can't remember)

**Comment by wohali** [comment]
Please try fully uninstalling CouchDB and reinstalling per these instructions (note, updated a few days ago):

https://docs.couchdb.org/en/latest/install/unix.html#installation-using-the-apache-couchdb-convenience-binary-packages

Your error is saying that the version of CouchDB installed does not match the version of OpenSSL on the system. You may have used the wrong build, i.e. our 18.04 or 20.04 version of CouchDB on 16.04.

We intend to support 16.04 for another year or so. You may prefer to try 18.04 or 20.04 for a better experience.
